packages/amplication-client/src/VersionControl/CommitsPage.tsx

```diff
@@ -52,15 +55,16 @@ const CommitsPage: React.FC<Props> = ({ match, moduleClass }) => {
         ) : null
       }
     >
-      {commitsBuilds ? (
+      {commitsBuildsState ? (
         <CommitResourceList
           builds={commits[commitIdx].builds}
           commitId={commitId}
         />
       ) : (
-        <div className="commit-page-empty-state">
+        <div className={`${moduleClass}__empty-state`}>
           <SvgThemeImage image={EnumImages.CommitEmptyState} />
-          <p>There are no commits to show</p>
+          {!commitsState && <p>There are no commits to show</p>}
+          {commitsState && !commitsBuildsState && <p>There are no builds to show</p>}
```

Comment:
I think it makes more sense to handle the empty state inside the CommitResourceList.
The component should be able to encapsulate the logic related to displaying the list, including when there are no items in the list. 

Also, it will make the code more readable and maintainable, especially if needed to use the same component elsewhere.